src/core/ui/components/user_account_settings/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Route } from '../../router/route';
+import { useRouter } from '../../router/context';
+import { EditNameScreen } from './screens/edit_name';
+import { SummaryScreen } from './screens/summary';
+import { Wrapper } from './layout/wrapper';
+import { VirtualRouter } from '../../router/virtual';
+import { useSlashauthClientUserAccountSettings } from './useSlashauthClientUserAccountSettings';
+import { useCallback } from 'react';
+import { LoginMethodType } from '../../context/login-methods';
+
+export const UserAccountSettings = withVirtualRouter(() => {
+  const {
+    accountSettings,
+    patchAccountSettings,
+    removeConnection,
+    addConnection,
+  } = useSlashauthClientUserAccountSettings();
+  const router = useRouter();
+
+  const addEmail = useCallback(() => {
+    addConnection([LoginMethodType.FederatedGoogle, LoginMethodType.MagicLink]);
+  }, [addConnection]);
+
+  const addWallet = useCallback(() => {
+    addConnection([LoginMethodType.Web3]);
+  }, [addConnection]);
+
+  // const addWeb2Account = useCallback(() => {
+  //   openSignIn({
+  //     connectAccounts: true,
+  //     includeLoginMethodTypes: [
+  //       LoginMethodType.FederatedDiscord,
+  //       LoginMethodType.FederatedTwitter,
+  //     ],
+  //   });
+  //   return;
+  // }, [openSignIn]);
```

Comment:
Should I add all web2Accounts related code @nedrocks ?

Reply:
hmm probably? What other web2 accounts do we have right now? I think none currently